.github/workflows/nightly.yml

```diff
@@ -0,0 +1,28 @@
+name: stale
+on:
+  schedule:
+    # runs every three days at 6am UTC
+    - cron: "0 6 */3 * *"
```

Comment:
[question] do the e2e tests take less than 24 hours to run? If yes, is it possible to run these every 24 hours?

Reply:
Yeah let's do it daily